styles/Vocab/Plone/reject.txt

```diff
@@ -1,5 +1,2 @@
-node
-nodejs
-javscript
-js
-Javascript
\ No newline at end of file
+[^.]js
```

Comment:
I don't understand this one. Reject any 3 letter word ending in js unless the first character is a period?

Reply:
Close. It matches words that end with "not-dot" followed by "js". Then reject the matches.

Thus it will match "Nodejs" and reject it, as the correct spelling is "Node.js".

side rant: Let it be known that punctuation in your branding and identity is a stupid idea. Wal-Mart is now Walmart, after they learned their lesson from bad SEO.